xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,112 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+int CGUIWindowVideoNav::SelectFirstUnwatchedItem()
+{
+  if (m_vecItems->IsVideoDb())
+  {
+    NODE_TYPE NodeType = CVideoDatabaseDirectory::GetDirectoryChildType(m_vecItems->GetPath());
```

Comment:
hmm that was a copy and paste from existing code in GUIViewStateVideo.cpp, line 118